### PR TITLE
Checking if the featured is supported before using

### PR DESCRIPTION
**Pull request by xiaodwan**
A temporary workaround for new features which doesn't
support in slow train

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_5.linux.latest8.arch_x86_64.it_vddk: PASS (327.86 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh please help to check this pr, thanks.

File: provider/v2v_vmcheck_helper.py [hunk 1]
```diff
@@ -93,9 +100,10 @@ def compare_version(self, compare_version):
         Compare virt-v2v version against given version.
         """
         cmd = 'rpm -q virt-v2v|grep virt-v2v'
-        v2v_version = LooseVersion(process.run(cmd, shell=True).stdout_text.strip())
+        v2v_version = LooseVersion(process.run(
+            cmd, shell=True).stdout_text.strip())
         compare_version = LooseVersion(compare_version)
-        if v2v_version > compare_version:
+        if v2v_version >= compare_version:
```
Comment (chloerh) [inline provider/v2v_vmcheck_helper.py]: There could be risk since the comparing strategy has changed.

**Comment by chloerh** [comment]
Could you remove the reformatting parts and only leave the necessary changes?

**Pull request merged**